tests/integration/rsi/test_rsi_change_answers.py

```diff
@@ -0,0 +1,159 @@
+                "metadata": {
+                    "user_id": "integration-test",
+                    "ru_ref": "123456789012A"
+                },
+                "data": {
+                    "11": "01/04/2016",
+                    "12": "30/04/2016",
+                    "20": "100",
+                    "21": "50",
+                    "146": "Down streams data test"
+                }
+            }
+        }
+
+        # Enable full dictionary diffs on test failure
+        self.maxDiff = None
+        self.assertDictEqual(actual, expected)
+
+    def test_happy_path_1_0112(self):
+        self.launchSurvey('1', '0112', roles=['dumper'])
+
+        # We are on the landing page
+        self.assertInPage('>Start survey<')
+        self.assertInPage('Monthly Business Survey - Retail Sales Index')
+
+        # We proceed to the questionnaire
+        self.post(action='start_questionnaire')
+
+        # We are in the Questionnaire
+        self.assertInPage('>Monthly Business Survey - Retail Sales Index</')
+        self.assertInPage('What are the dates of the period that you will be reporting for?')
+        self.assertInPage('>Save and continue<')
+
+        # When I submit an answer
+        form_data = {
+            # Start Date
+            'period-from-day': '01',
+            'period-from-month': '4',
+            'period-from-year': '2016',
+            # End Date
+            'period-to-day': '30',
+            'period-to-month': '4',
+            'period-to-year': '2016',
+        }
+
+        # We submit the form
+        self.post(form_data)
+        self.post(post_data={'total-retail-turnover-answer': '100'})
+        self.post(post_data={'internet-sales-answer': '50'})
+        self.post(post_data={'changes-in-retail-turnover-answer': 'Down streams data test'})
+        self.post(post_data={'male-employees-over-30-hours': '1'})
+        self.post(post_data={'male-employees-under-30-hours': '2'})
+        self.post(post_data={'female-employees-over-30-hours': '3'})
+        self.post(post_data={'female-employees-under-30-hours': '4'})
+        self.post(post_data={'total-number-employees': '5'})
+        self.post(post_data={'changes-in-employees-answer': 'change in number of employees'})
+
+        # There are no validation errors (we're on the summary screen)
+        self.assertInUrl('summary')
+        self.assertInPage('>Monthly Business Survey - Retail Sales Index</')
+        self.assertInPage('>Your responses<')
+        self.assertInPage('Please check carefully before submission')
+        self.assertInPage('>Submit answers<')
+
+        # And the JSON response contains the data I submitted
+        actual = self.dumpSubmission()
+
+        expected = {
+            "submission": {
+                "data": {
+                    "11": "01/04/2016",
+                    "12": "30/04/2016",
+                    "20": "100",
+                    "21": "50",
+                    "50": "5",
```

Comment:
I don't think we submit those fields to downstream as they do not have q_codes. I will double check with jenny